apps/authentication-portal/src/main/webapp/totp.jsp

```diff
@@ -45,8 +47,20 @@
 
     if (Boolean.parseBoolean(request.getParameter(Constants.AUTH_FAILURE))) {
         authenticationFailed = "true";
-
-        if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
+        String errorCode = request.getParameter(TOTPAuthenticatorConstants.ERROR_CODE);
+
+        if (errorCode != null) {
+            if (errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE)) {
```

Comment:
what would happen if `errorCode != null && !errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE)`

Reply:
That would display the `Authentication Failed! Please Retry` message. as set in https://github.com/wso2/identity-apps/pull/3502/files#diff-d1b2fde3cd88b915b5a32dd5848232edd132779bc93f21a22ec7ae2f4c8d64b7R45

Reply:
However its better to go with previous flow if error message is not added from error code.
This is addressed in dc536c73596f9c931adb6401604f2ecd56ff0eec